src/slim-select/store.ts

```diff
@@ -253,29 +259,6 @@ export default class Store {
     }, false) as Option[]
   }
 
-  public getSelectedIDs(): string[] {
```

Comment:
You probably removed cause they arent use internally but they are public and could be used by others. Lets leave them in for now and if i do a breaking release I can consider removing them then.

Reply:
Yeah, removing stuff from a public API is pretty stupid. No idea why I thought this was a good idea 😅